app/helpers/sessions_helper.rb

```diff
@@ -1,2 +1,5 @@
 module SessionsHelper
+	def current_user_can_edit_post
+		current_user and current_user.can_edit? post
+	end
```

Comment:
Not the correct indentation. Did you run rubocop?!

Reply:
We will run rubocop after the merge, or resolve any conflicts before that.